web/tests/search.test.js

```diff
@@ -331,12 +331,11 @@ run_test("initiate_search", () => {
         is_searchbox_text_selected = true;
     });
 
+    narrow_state.filter = () => ({is_search: () => false});
     $(".navbar-search.expanded").length = 0;
     search.initiate_search();
     assert.ok(typeahead_forced_open);
     assert.ok(is_searchbox_text_selected);
     // test that we append space for user convenience
-    narrow_state.filter = () => ({is_search: () => false});
-    search.initiate_search();
```

Comment:
It doesn't make sense to `search.initiate_search();` while it's already open -- to test for the appended space, we need to initiate search from a closed search state, which we were already doing.

So I updated the test to only initiate search once